resources/scripts/ttx_diff.py

```diff
@@ -616,6 +610,19 @@ def delete_things_we_must_rebuild(rebuild: str, fontmake_ttf: Path, fontc_ttf: P
                     os.remove(path)
 
 
+# returns the path to the compiled binary
+def build_crate(manifest_path: Path):
+    cmd = ['cargo', 'build', '--release', '--manifest-path', str(manifest_path)]
+    run(cmd, None, check=True)
```

Comment:
this isn't calling subprocess.run, it's calling a run fn defined in this file.

Reply:
ahah bad choice of name then 😅

Reply:
if it's passing working_dir=None and the entire reason of existence of ttx_diff.run() is to call subprocess.run with the given cwd=working_dir, then it's better to avoid the wrapper altogether and simply use subproces.run directly?

Reply:
I think the main purpose is that it also logs the command before running.